scalafix/rules/src/main/scala/fix/Roughly.scala

```diff
@@ -48,6 +48,10 @@ final case class Roughly(index: SemanticdbIndex, config: RoughlyConfig)
     Symbol("_root_.scala.collection.immutable.Stream.append.")
   )
 
+  val streamEmpty = SymbolMatcher.exact(
+    Symbol("_root_.scala.collection.immutable.Stream.Empty.")
```

Comment:
Can you add it to the scalafix 0.6 branch?

```scala
val streamEmpty = exact(
  "scala/collection/immutable/Stream#empty()."
)
```

https://github.com/scala/scala-collection-compat/blob/scalafix-0.6/scalafix/rules/src/main/scala/fix/Roughly.scala#L44-L46